public/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Kytra Live CSS</title>
+
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Lato|PT+Sans|Roboto+Condensed&display=swap" rel="stylesheet">
```

Comment:
Since the chrome extension can be used while offline, it's better practice to store this locally rather than rely on a CDN